src/rleCompress.js

```diff
@@ -18,7 +19,22 @@
  * @return {string}
  */
 function rleCompress(source) {
-  // write code here
+  let compressedString = '';
+  let counter = 1;
+
+  for (let i = 0; i < source.length; i++) {
+    if (source[i] === source[i + 1]) {
+      counter += 1;
+    } else if (counter > 1) {
+      compressedString += (source[i]);
```

Comment:
remove brackets everywhere
```suggestion
      compressedString += source[i];
```